components/report-details/support/dialog.tsx

```diff
@@ -140,25 +143,45 @@ const SupportReport = ({
 		setOpen,
 	};
 
-	if (isDesktop) {
-		return (
-			<ReportSupportUI
-				{...props}
-				UIComponent={Dialog}
-				ContentComponent={DialogContent}
-				HeaderComponent={DialogHeader}
-			/>
-		);
-	}
+	// if (isDesktop) {
+	// 	return (
+	// 		<ReportSupportUI
+	// 			{...props}
+	// 			UIComponent={Dialog}
+	// 			ContentComponent={DialogContent}
+	// 			HeaderComponent={DialogHeader}
+	// 		/>
+	// 	);
+	// }
 
+	// return (
+	// 	<ReportSupportUI
+	// 		{...props}
+	// 		UIComponent={Drawer}
+	// 		ContentComponent={(props) => (
+	// 			<DrawerContent {...props} className="bg-vd-beige-100 p-3" />
+	// 		)}
+	// 		HeaderComponent={DrawerHeader}
+	// 	/>
+	// );
 	return (
-		<ReportSupportUI
-			{...props}
-			UIComponent={Drawer}
-			ContentComponent={(props) => (
-				<DrawerContent {...props} className="bg-vd-beige-100 p-3" />
-			)}
-			HeaderComponent={DrawerHeader}
+		<DooglyDonateButton
+			buttonText="Support this report"
+			modalTitle="Support this report"
+			config={{
+				destinationChain: "optimism",
+				destinationAddress: "0x8a4c14d50c43363a28647188534db7004112091c",
```

Comment:
It is probably a good idea to move the addresses to const and add a comment on what they are for.

Reply:
Updated!